src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -6,9 +6,9 @@
 import net.minecraft.stats.Achievement;
 
 /**
- * A sublcass for this mod.
+ * A subclass for this mod.
```

Comment:
Please improve this description. What does it contain (maybe doesnt count, what is actually there)? What is it's purpose? (Here you can add maybes as "We want this to cntain X")